src/eigen/preconditioners.jl

```diff
@@ -67,6 +68,7 @@ ldiv!(P::PreconditionerTPA, R) = ldiv!(R, P, R)
     Y
 end
 mul!(P::PreconditionerTPA, R) = mul!(R, P, R)
```

Comment:
There is no single-argument mul! in LinearAlgebra. I don't think we should define that ... (note: It's also not used in IterativeSolvers.jl)